core/src/manager/tab.rs

```diff
@@ -258,6 +258,10 @@ impl Tab {
 		};
 		true
 	}
+
+	pub fn current_name(&self) -> &str {
+		self.current.cwd.file_name().and_then(|name| name.to_str()).unwrap_or_default()
```

Comment:
Using the `cwd` itself instead of the default might be better when can't get the filename.

Reply:
Do you mean return `self.current.cwd`?

Reply:
Like this https://github.com/sxyazi/yazi/blob/866ed1704459b91b7e6ea220e4aadcbd1397ba6d/config/src/pattern.rs#L23